lib/ridgepole/ext/abstract_mysql_adapter.rb

```diff
@@ -1,16 +1,26 @@
 require 'active_record/connection_adapters/abstract_mysql_adapter'
 
-class ActiveRecord::ConnectionAdapters::AbstractMysqlAdapter
-  def add_index_with_alter(table_name, column_name, options = {})
-    index_name, index_type, index_columns, index_options, index_algorithm, index_using = add_index_options(table_name, column_name, options)
+module Ridgepole
+  module Ext
+    module AbstractMysqlAdapter
+      def add_index(table_name, column_name, options = {})
+        index_name, index_type, index_columns, index_options, index_algorithm, index_using = add_index_options(table_name, column_name, options)
 
-    # cannot specify index_algorithm
-    execute "ALTER TABLE #{quote_table_name(table_name)} ADD #{index_type} INDEX #{quote_column_name(index_name)} #{index_using} (#{index_columns})#{index_options}"
+        # cannot specify index_algorithm
+        execute "ALTER TABLE #{quote_table_name(table_name)} ADD #{index_type} INDEX #{quote_column_name(index_name)} #{index_using} (#{index_columns})#{index_options}"
+      end
+
+      def remove_index!(table_name, index_name)
+        execute "ALTER TABLE #{quote_table_name(table_name)} DROP INDEX #{quote_column_name(index_name)}"
+      end
+    end
   end
-  alias_method_chain :add_index, :alter
+end
 
-  def remove_index_with_alter!(table_name, index_name)
-    execute "ALTER TABLE #{quote_table_name(table_name)} DROP INDEX #{quote_column_name(index_name)}"
+module ActiveRecord
+  module ConnectionAdapters
+    class AbstractMysqlAdapter
+      include Ridgepole::Ext::AbstractMysqlAdapter
```

Comment:
Fixed https://github.com/winebarrel/ridgepole/pull/68
Thank you for your comment :bow: